letsencrypt/cli.py

```diff
@@ -621,7 +655,7 @@ def create_parser(plugins, args):
         help="show program's version number and exit")
     helpful.add(
         "automation", "--no-confirm", dest="no_confirm", action="store_true",
```

Comment:
Can we please rename it to something less vague? How about `--renew-by-default`? Also, you can remove `dest`.